Gemfile.lock

```diff
@@ -168,6 +168,7 @@ GEM
       date
       net-protocol
     net-pop (0.1.2)
+      net-protocol
```

Comment:
This missing dependency of `net-pop` is why the deploy was failing with Ruby 3.3.3.